src/signing-panacea-client.ts

```diff
@@ -239,4 +246,40 @@ export class SigningPanaceaClient extends SigningStargateClient {
     };
     return this.signAndBroadcast(ownerAddress, [msg], this.panaceaFees.issueToken, memo);
   }
+
+  async createDeal(ownerAddress: string, dataSchema: string[], budget: Coin, maxNumData: Long, trustedDataValidators: string[], memo?: string): Promise<DeliverTxResponse> {
+    const msg = {
+      typeUrl: SigningPanaceaClient.msgTypeCreateDeal,
+      value: {
+        owner: ownerAddress,
+        dataSchema: dataSchema,
+        budget: budget,
+        maxNumData: maxNumData,
+        trustedDataValidators: trustedDataValidators,
```

Comment:
These values are not ordered as defined in `.proto` file.
However it doesn't matter since the test for createDeal has been passed.
Any idea?